docs/pages/versions/unversioned/sdk/asset.md

```diff
@@ -115,16 +115,16 @@ Returns a Promise that resolves with an array of `Asset`s when the asset(s) has
 #### Example
 
 ```ts
-const [{ localUri }] = await Asset.loadAsync(require('./assets/snack-icon.png'));
+const [{ localUri }] = await Asset.loadAsync('https://docs.expo.io/static/images/header-logo.png');
```

Comment:
I think it's better to leave the example as it was — requiring a local asset is a more common use case 🙂 
```suggestion
const [{ localUri }] = await Asset.loadAsync(require('./assets/snack-icon.png'));
```